locales/en-US/app.l20n

```diff
@@ -104,8 +103,9 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainer "There are <span data-l10n-id="userCount" class="bold" data-hook="install-count"></span> people trying <span data-hook="title"></span> right now!">
```

Comment:
Hmm, this is not quite working. Two problems:

1) The l20n string needs to have the quotes escaped.
2. Localized string replacement happens *after* Ampersand renders, so it never gets a chance to populate these `data-hook` elements.

So, you need something more like this:
```
<userCountContainer "There are <span class=\"bold\">{{installation_count}}</span> people trying {{title}} right now!">
```